leabra7/net.py

```diff
@@ -156,11 +185,37 @@ def new_projn(self,
 
         pre_lr = self._get_layer(pre)
         post_lr = self._get_layer(post)
-        pr = projn.Projn(name, pre_lr, post_lr, spec)
+        pr = projn.Projn(name, pre_lr, post_lr, spec=spec)
         self.projns[name] = pr
         self.objs[name] = pr
         self._add_loggers(pr)
 
+    def inhibit_projns(self, *projn_names: str) -> None:
```

Comment:
this is a nice syntax. if you are going to use it you should change the force method (and other similar ones on the net class) to use the same interface, to keep things consistent.

Reply:
Implemented with pause/resume logging and unclamping and projection and layer validation